### ISSUE TITLE
Automattic deployments are not configured

### ISSUE THREAD

**Issue by christianc1** [ChurchThemes-WP/shepherd-theme#22]
With all of our themes, commits to master are auto-deployed to our servers.  This doesn't appear to be the case with this theme.  Can you please provide some clarification?

**Comment by chriswallace** [comment]
I think this one is served from WordPress.org. We have to submit a new version to their review team to get it updated.

**Comment by angela-crist** [comment]
@christianc1 Where do we stand here, do we all still need to meet?

**Comment by christianc1** [comment]
No, however I may have to be added as an author, I'm not sure exactly how that works.  Attn: @chriswallace

**Comment by angela-crist** [comment]
Has this been resolved @christianc1
